tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -92,6 +151,13 @@ impl Json {
     pub fn with_span_list(&mut self, display_span_list: bool) {
         self.display_span_list = display_span_list;
     }
+
+    /// If set to `false`, formatted events won't be followed by a newline.
+    /// This option is mainly useful for logic that is supposed to expand logged
+    /// JSON values by embedding them in a wrapping JSON structure.
```

Comment:
maybe this should link to the docs section in the type-level docs you added on how to wrap the JSON formatter?